pkg/hookbot/hookbot.go

```diff
@@ -308,3 +308,19 @@ func (h *Hookbot) ServeSubscribe(conn *websocket.Conn, r *http.Request) {
 		}
 	}
 }
+
+func IsConnectionClose(err error) bool {
```

Comment:
I've done this tens of times now. This is the least horrible solution that reliably works.